apps/avifgainmaputil/combine_command.cc

```diff
@@ -98,13 +98,11 @@ avifResult CombineCommand::Run() {
   }
 
   const int downscaling = std::max<int>(1, arg_downscaling_);
-  const uint32_t gain_map_width = std::max(
-      static_cast<uint32_t>(std::round((float)base_image->width / downscaling)),
-      1u);
+  const int rounding = downscaling / 2;
+  const uint32_t gain_map_width =
+      std::max((base_image->width + rounding) / downscaling, 1u);
```

Comment:
`width` is `uint32_t`, `rounding` and `downscaling` should be unsigned too, no?

Reply:
Done. When `int` and `unsigned int` are used in the same operation, `int` is implicitly converted to `unsigned int`. It's good to use the desired type explicitly.

Reply:
I remember compiler warnings on Windows complaining about signed/unsigned mismatch. I understand it is fine according to the C standard though.

Reply:
Compilers warn about signed/unsigned mismatch in comparisons. I have not seem compiler warnings about signed/unsigned mismatch in arithmetic operations (`+`, `-`, `*`, `/`).